src/const/partnerRewardsTheme.ts

```diff
@@ -8,16 +8,32 @@ export const PROFILE_CAMPAIGN_LIGHT_TOKEN = `${process.env.NEXT_PUBLIC_STRAPI_UR
 export const PROFILE_CAMPAIGN_DARK_CHAIN = `${process.env.NEXT_PUBLIC_STRAPI_URL}/uploads/Sei_Symbol_Gradient_ea276889b3.png`;
 export const PROFILE_CAMPAIGN_LIGHT_CHAIN = `${process.env.NEXT_PUBLIC_STRAPI_URL}/uploads/Sei_Symbol_Gradient_ea276889b3.png`;
 
-export const PROFILE_CAMPAIGN_SCANNER = 'https://seitrace.com';
-
-export const REWARDS_DECIMALS = 1;
-
-export const REWARD_TOKEN_CHAINID = 1329;
-export const REWARD_TOKEN_ADDRESS =
-  '0xE30feDd158A2e3b13e9badaeABaFc5516e95e8C7';
-export const REWARD_CLAIMING_ADDRESS =
-  '0x3Ef3D8bA38EBe18DB133cEc108f4D14CE00Dd9Ae';
-
-export const REWARDS_CHAIN_IDS = ['1329'];
+export const REWARDS_LIST = [
+  {
+    chainId: '1329',
+    tokenChainId: 1329,
+    claimingAddress: '0x3Ef3D8bA38EBe18DB133cEc108f4D14CE00Dd9Ae',
+    tokenAddress: '0xE30feDd158A2e3b13e9badaeABaFc5516e95e8C7',
+    decimalsToShow: 1,
+    explorerLink: 'https://seitrace.com',
+    lightTokenLogo: `${process.env.NEXT_PUBLIC_STRAPI_URL}/uploads/Sei_Symbol_Gradient_ea276889b3.png`,
```

Comment:
The images are the same for all the TokenLogo and ChainLogo keys, is that expected? if yes it can be refactorized